src/database/src/js/postGresOperations.js

```diff
@@ -0,0 +1,214 @@
+ * name/value pair can be omitted.
+ *
+ * @param {Object} that - Postgres request object.
+ * @param {Sting} tableName - Name of table to insert into.
+ * @param {Array} jsonArray - An array of JSON objects to load.
+ * @return {Promise} whose value is an array of successful INSERT results.
+ */
+fluid.postgresdb.loadFromJSON = function (that, tableName, jsonArray) {
+    var insertions = [];
+    fluid.each(jsonArray, function (aRecord) {
+        var columns = fluid.postgresdb.stringifyJoinKeys(Object.keys(aRecord));
+        var values = fluid.postgresdb.stringifyJoinValues(Object.values(aRecord));
+        insertions.push(
+            `INSERT INTO "${tableName}" (${columns}) VALUES (${values});`
+        );
+    });
+    // For debugging
+    fluid.each(insertions, function (anInsertion) {
+        fluid.log(fluid.logLevel.TRACE, anInsertion);
+    });
+    return that.bulkQuery(insertions);
+};
+
+/**
+ * Utility to stringify each element of an array of object keys and join them
+ * as a single string separated by commas.  This is similar to the Array.join()
+ * function, but here, each element is quoted with double quotes.
+ *
+ * @param {Array} keys - Array of object keys.
+ * @return {String} - The keys in a comma separated string where each key is
+ *                    quoted.
+ */
+fluid.postgresdb.stringifyJoinKeys = function (keys) {
+    if (keys.length === 0) {
+        return "";
+    }
+    var theJoin = JSON.stringify(keys[0]);
+    for (var i = 1; i < keys.length; i++) {
+        theJoin += "," + JSON.stringify(keys[i]);
+    }
+    return theJoin;
+};
+
+/**
+ * Utility to stringify each element of an array of object values and join them
+ * as a single string separated by commas.  This is similar to the Array.join()
+ * function, but where each element is quoted with a single quote.  If a value
+ * is a plaing object, it is stringified as well as quoted.
+ *
+ * @param {Array} values - Array of object values.
+ * @return {String} - The keys in a comma separated string where each key is
+ *                    quoted.
+ */
+fluid.postgresdb.stringifyJoinValues = function (values) {
+    if (values.length === 0) {
+        return "";
+    }
+    var theJoin = "'" + fluid.postgresdb.maybeStringify(values[0]) + "'";
+    for (var i = 1; i < values.length; i++) {
+        theJoin += ",'" + fluid.postgresdb.maybeStringify(values[i]) + "'";
+    }
+    return theJoin;
+};
+
+/**
+ * Utility to return a stringified object if given an object; otherwise just
+ * return it.
+ *
+ * The input value is checked via `fluid.isPrimitive()`, and if so, it is
+ * returned as is.  Otherwise the result of `JSON.stringify()` is returned. If
+ * the argument is an array, the square brackets are replaced with curly ones.
+ * @param {Any} aValue - Value to stringify if necessary.
+ * @return {Any|String} - The value as is, or stringified.
+ */
+fluid.postgresdb.maybeStringify = function (aValue) {
```

Comment:
Conditional conversion functions like this are a code smell and likely an injection risk. It should be the responsibility of the caller to convert data into a form which is already suitable to be inserted into an SQL table rather than relying on some implicit expectations.